Update Continuous Testing global configuration docs (#15992)
DataDog/documentation: content/en/continuous_testing/cicd_integrations/configuration.md

    datadog-ci synthetics run-tests --apiKey "<API_KEY>" --appKey "<APPLICATION_KEY>"
    ```

> 3. Or defined in a [global configuration fil](#global-configuration-file-options):

Reviewer: ```suggestion
3. Or defined in a [global configuration file](#global-configuration-file-options):
```

`datadogSite`
: The Datadog instance to which request is sent. The default is `datadoghq.com`. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.

`failOnCriticalErrors`
: A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.

`failOnMissingTests`
: A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.

`failOnTimeout`
: A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.

> 

Reviewer 2: We can add `pollingTimeout` in here as well if it helps making it more evident (we support both `config.pollingTimeout` & `config.global.pollingTimeout`)

`datadogSite`
: The Datadog instance to which request is sent. The default is `datadoghq.com`. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.

`failOnCriticalErrors`

> : A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.

Reviewer: ```suggestion
: A boolean flag that fails the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
```

`failOnMissingTests`

> : A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.

Reviewer: ```suggestion
: A boolean flag that fails the CI job if at least one test is missing in a run (for example, if it has been removed or deleted). The default is set to `false`.
```

`failOnTimeout`

> : A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.

Reviewer: ```suggestion
: A boolean flag that fails the CI job if at least one test exceeds the default test timeout. The default is set to `true`.
```

`files`
: Glob pattern to detect Synthetic tests config files.

`global`

> : Overrides of Synthetic tests applied to all tests ([see below for descriptions of each field](#additional-configuration)).

Reviewer: Which header does this anchor link to?

```suggestion
: Overrides of Synthetic tests applied to all tests. [See below for descriptions of each field]().
```

Author: @alai97 Do we want to remove the link to the `additional configuration` section?
